src/shared/dissect-image.c

```diff
@@ -1661,6 +1701,40 @@ int dissect_image_and_warn(
         }
 }
 
+bool dissected_image_can_do_verity(const DissectedImage *image, unsigned partition_number) {
```

Comment:
"partition_number" is a misnomer. It's a designator type, not a partition index after all

Reply:
changed to partition_deisgnator